internal/junit/reporter.go

```diff
@@ -25,6 +27,99 @@ func (r *Reporter) Add(t report.TestResult) {
 	r.TestResults = append(r.TestResults, t)
 }
 
+func parseJunitFiles(junits []report.Artifact) ([]TestSuites, error) {
+	var parsed []TestSuites
+	var errs []error
+	for _, ju := range junits {
+		if ju.Error != nil {
+			errs = append(errs, errors.New("failed to retried junit file"))
+			continue
+		}
+		ts, err := Parse(ju.Body)
+		if err != nil {
+			errs = append(errs, errors.New("failed to parse junit file"))
+			continue
+		}
+		parsed = append(parsed, ts)
+	}
+	if len(errs) > 0 {
+		return parsed, fmt.Errorf("%d errors occured while evaluating junit files", len(errs))
```

Comment:
Where does the user, or we, if we have to troubleshoot, see what's wrong with them? This will just end up as a support ticket "because saucectl failed to download my reports".

Reply:
I added wrapped errors